src/components/ExpandableCard.js

```diff
@@ -81,7 +81,7 @@ const StyledFakeLink = styled(FakeLink)`
   white-space: nowrap;
 `
 
-const ExpandableCard = ({ children, contentPreview, title, svg, alt }) => {
+const ExpandableCard = ({ children, contentPreview, title, Svg, alt }) => {
```

Comment:
I would suggest using lowercase for props to follow conventions. Then you can redefine it
```jsx
const Svg = props.svg
```
https://airbnb.io/javascript/react/#props